packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1212,15 +1212,9 @@ export class ContainerRuntime
 			() => this.storage,
 			(localId: string, blobId?: string) => {
```

Comment:
Can you add a UT to repro the bug/fix?

Reply:
This is mainly reverting a change that was part of the grouped batching PR.
I can create a follow up for UT as grouped batching option is still considered experimental.